### ISSUE TITLE
Does not backup Hangouts / Project-Fi messages

### ISSUE THREAD

**Issue by redstone99** [jberkel/sms-backup-plus#698]
After switching to Project-Fi, this app stopped backing up SMS messages (automatically or manually).  When I click 'backup', it says 'there are no items to backup'.  I use hangouts for SMS, so that may be related to the issue.  Android 7.0 on Nexus 5x.

**Comment by chipmunkofdoom2** [comment]
Same issue. Nexus 5x, Project Fi, Android 7.0.

I noticed something interesting and connected some dots using this thread on Reddit. I have a few SMS that have a grey tag in Hangouts. Apparently, this means the SMS was sent through the carrier. When the SMS has a blue tag, this means it was sent from Google Voice. All the "blue" tag messages are not backing up, while the few grey tag messages are:

https://www.reddit.com/r/ProjectFi/comments/3hls8o/mms_picture_messages_wont_send_to_some_people/

So long story short, I think the problem is on Fi with Hangouts, Google Voice's servers send the texts over data in these cases.

**Comment by jberkel** [comment]
yes, non-standard apps aren't supported: https://github.com/jberkel/sms-backup-plus#only-received-messages-are-backed-up-not-the-ones-i-sent

Duplicate of #516
